src/js/facility-locator/components/FacilityMap.jsx

```diff
@@ -17,6 +17,7 @@ class HowToGetHere extends Component {
       va_health_facility: 'health',
       va_cemetery: 'cemetery',
       va_benefits_facility: 'benefits',
+      vet_center: 'vet-centers',
```

Comment:
Would it be easier to maintain this by transforming the values of `facilityTypes` from `config` using `_.kebabCase`?